roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/entity/StreetSectionTest.java

```diff
@@ -58,7 +83,34 @@ public void firstCarCouldEnterNextSection() throws Exception {
     }
 
     @Test
-    public void isEnoughSpace() throws Exception {
+    public void firstCarCouldEnterNextSection_everythingOK() throws Exception {
+        IStreetSection streetSectionMock = mock(StreetSection.class);
+        when(streetSectionMock.firstCarCouldEnterNextSection()).thenCallRealMethod();
+        when(streetSectionMock.isFirstCarOnExitPoint()).thenReturn(true);
+        ICar firstCar = mock(Car.class);
+        when(streetSectionMock.getFirstCar()).thenReturn(firstCar);
+        IStreetSection nextStreetSection = mock(StreetSection.class);
+        when(firstCar.getNextStreetSection()).thenReturn(nextStreetSection);
+        when(firstCar.getLength()).thenReturn(5.0);
+        when(nextStreetSection.isEnoughSpace(firstCar.getLength())).thenReturn(true);
+        IStreetConnector streetConnector = mock(StreetConnector.class);
+        when(streetSectionMock.getPreviousStreetConnector()).thenReturn(streetConnector);
+        HashSet<IStreetSection> precendenceSections = new HashSet<>();
+        IStreetSection streetSectionOne = mock(StreetSection.class);
+        when(streetSectionOne.isFirstCarOnExitPoint()).thenReturn(false);
+        IStreetSection streetSectionTwo = mock(StreetSection.class);
+        when(streetSectionTwo.isFirstCarOnExitPoint()).thenReturn(false);
+        precendenceSections.add(streetSectionMock);
+        precendenceSections.add(streetSectionOne);
+        precendenceSections.add(streetSectionTwo);
+        when(streetConnector.getPreviousSections()).thenReturn(precendenceSections);
+
+        // everything ok
+        assertTrue(streetSectionMock.firstCarCouldEnterNextSection());
+    }
+
+    @Test
+    public void isEnoughSpace_bigger() throws Exception {
```

Comment:
nobody knows what "bigger" in this given context means
"spaceIsBiggerThanCar" or so... you know better what this test does